charts/statping/values.yaml

```diff
@@ -1,171 +1,83 @@
-# Default values for statping.
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/charts/tree/master/charts/common/values.yaml
+#
 
 image:
   repository: statping/statping
   pullPolicy: IfNotPresent
-  tag: ""
+  tag: "v0.90.65"
+
+strategy:
+  type: Recreate
+
+# See more environment variables in the statping documentation
+# https://github.com/statping/statping/wiki/Config-with-.env-File
+env:
+  # -- Name of the Statping instance
+  NAME: Statping Example
+  # -- Description of the Statping instance
+  DESCRIPTION: This is a Statping instance deployed as Helm chart
+  # -- External URL you use to connect to the statping (the one you enter in your browser)
+  VIRTUAL_HOST: ""
+  # -- Type of database to use
+  DB_CONN: postgres
+  # -- Postgres database hostname
+  # @default -- internal postgresql URL
+  DB_HOST: ""
+  # -- Enable ssl_mode for postgres (To enable use require)
+  POSTGRES_SSLMODE: "disable"
+  # -- Postgres database user name
+  # @default -- postgresql.postgresqlUsername value
+  DB_USER: ""
+  # -- Postgres database password
+  # @default -- postgresql.postgresqlPassword value
+  DB_PASS: ""
+  # -- Postgres database password
+  # @default -- postgresql.postgresqlDatabase value
+  DB_DATABASE: ""
+  # -- dashboard admin user (needed to edit)
+  ADMIN_USER: admin
+  # -- dashboard admin password (needed to edit)
+  ADMIN_PASSWORD: admin
+  # -- dashboard admin email
+  ADMIN_EMAIL: info@example.com
+  # -- Use CDN for static context from third-parties
+  USE_CDN: false
+  # -- Disable logs from appearing and writing to disk
+  DISABLE_LOGS: false
```

Comment:
Please replace the `# --` with `##` (also in the rest of the file) to keep the comments consistent with the other charts

Reply:
I think the comment format are for helmdocs

Reply:
I tried but then the descriptions were not recognized. Is there a new version/task to use for `##` to work with helm-docs?

Reply:
@angelnu I think you have it right, it's something that should be updated across our charts so helm docs can pick up the descriptions